apps/files_sharing/src/views/SharingTab.vue

```diff
@@ -264,7 +264,7 @@ export default {
 				const shares = data.ocs.data
 					.map(share => new Share(share))
 					.sort((a, b) => {
-						const localCompare = a.title.localeCompare(b.title)
+						const localCompare = a.shareWithDisplayName.localeCompare(b.shareWithDisplayName)
```

Comment:
We could do something like except labels are only available for link shares and links are display before all other shares without any mixing up. The best way then is to use the receiver display names for the sorting.

```suggestion
						const labelA = a.label || a.shareWithDisplayName;
						const labelB = b.label || b.shareWithDisplayName;
						const localCompare = labelA.localeCompare(labelB)
```